cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	"sigs.k8s.io/kubebuilder/pkg/model/config"
+
+	genutil "github.com/operator-framework/operator-sdk/cmd/operator-sdk/generate/internal"
+	gencsv "github.com/operator-framework/operator-sdk/internal/generate/clusterserviceversion"
+	"github.com/operator-framework/operator-sdk/internal/generate/collector"
+)
+
+func (c *bundleCmd) setDefaults(cfg *config.Config) {
+	if c.operatorName == "" {
+		c.operatorName = filepath.Base(cfg.Repo)
+	}
+}
+
+func (c bundleCmd) runKustomize(cfg *config.Config) error {
```

Comment:
```suggestion
// runKustomize will run the `kustomize build` on those directories. Note that kustomize commands are also used 
// in the Makefile targets. The `make bundle` runs `kustomize build` on those directories in the case of any change be applied (ex. patches) wanted by a user to their manifests occur before being read by generate a bundle.
func (c bundleCmd) runKustomize(cfg *config.Config) error {
```

Reply:
You will be able to come up with a better explanation :-)

Reply:
Done.